login/index.js

```diff
@@ -0,0 +1,61 @@
+/* 
+- 이메일 input에서 focus out 할 때, 값이 없을 경우 아래에 “이메일을 입력해 주세요.” 에러 메세지를 보입니다.
+- 비밀번호 input에서 focus out 할 때, 값이 없을 경우 아래에 “비밀번호를 입력해 주세요.” 에러 메세지를 보입니다.
+- 이메일 input에서 focus out 할 때, 이메일 형식에 맞지 않는 값이 있는 경우 아래에 “올바른 이메일 주소가 아닙니다.” 에러 메세지를 보입니다.
+- 이메일: test@codeit.com, 비밀번호: codeit101 으로 로그인 시도할 경우, “/folder” 페이지로 이동합니다.
+- 이외의 로그인 시도의 경우, 이메일 input 아래에 “이메일을 확인해 주세요.”, 비밀번호 input 아래에 “비밀번호를 확인해 주세요.” 에러 메세지를 보입니다.
+- input에 에러와 관련된 디자인은 좌측 상단의 Components 영역에 있는 디자인을 참고해 주세요.
+- 로그인 버튼 클릭 또는 Enter키 입력으로 로그인 실행돼야 합니다.
+*/
+
+function createErrorMessage(message) {
+  let errorMessage = document.createElement('div');
+  errorMessage.textContent = message;
+  errorMessage.className = 'error-message';
+  return errorMessage;
+}
+
+function appendErrorMessage(errorContainer, message) {
+  errorContainer.innerHTML = message;
+}
+
+function showErrorMessage(emailErrorMessage, passwordErrorMessage) {
+  let emailInput = document.querySelector('.sign-input');
+  let passwordInput = document.querySelector('sign-input:nth-child(2)');
```

Comment:
@hoonj170214 

```suggestion
  const emailInputEl = document.querySelector('input[type="email"]');
  const passwordInputEl = document.querySelector('input[type="password"]');
```


1. 변경되지 않을 요소의 경우, const로 변수 선언을 해주세요!
2. `input:nth-child`보다, 해당 Input이 지닌 type 프로퍼티 기준으로 선택자를 활용하는게 좋아보여요!

Reply:
1. const 로 수정했습니다 
2. 이 부분에서 오류가 났었는데, `nth-child` 로 해서 그랬었네요. 감사합니다 👍

Reply:
`const`가 아닌 `let` 으로 변수를 선언했던 이유는 
- const 는 변경(재할당)이 불가능한 상수
- `emailInput` , `passwordInput` 변수에 이벤트 핸들러를 등록하려면 변경(?)이 일어난다
- 따라서 `const`가 아닌 `let`으로 변경(재할당)할 수 있는 변수로 선언해야 한다
고 생각했었습니다. 일단 멘토님이 알려주신대로 수정 후에 다시 복습을 해보니 

그런데 제가 간과하고 있었던 부분이
- `const`로 선언된 변수는 재할당이 불가능하지만,  해당 변수가 참조하는 객체(DOM요소)의 내용은 변경될 수 있다.
- 변수가 참조하는 객체가 변하지 않는다는 `const`의 제약을 지키면서도 객체 내부의 속성 등은 변경될 수 있다.
였습니다. 

기본을 생각하지 못하고 있었네요ㅠ  알려주셔서 감사합니다 👍

Reply:
@hoonj170214 

오 제대로 이해하셨습니다!
잘 하고 계시니 너무 걱정 마셔요 ㅋㅋ